calicoctl/commands/build_info.go

```diff
@@ -15,8 +15,4 @@
 // WARNING: This file is overwritten by a script in the release build process.
 package commands
 
-const (
```

Comment:
We can probably remove this entire file, and just stick these vars in version.go rather than splitting up the different info.  These are only used in version.go, I think.